CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # CHANGELOG
 
-* [1.0.0](#0.0.1)
+* [1.0.1](#1.0.1)
```

Comment:
If the version has new features I think we should use a minor number instead of a patch number.

Reply:
Changed